kernel/kevinfs.c

```diff
@@ -142,16 +145,20 @@ static int kevinfs_ata_write_superblock(struct device *device)
 	uint32_t counter = 0;
 	printf("Writing inode bitmap...\n");
 	for (uint32_t i = super.inode_bitmap_start; i < super.inode_start; i++) {
-		if(!bcache_write(device, zeros, 1, i))
+		if(!bcache_write(device, zeros, 1, i)) {
```

Comment:
Careful with failure case here: bcache_write returns number of blocks (1) on success, and negative for failure.